django_comments/views/comments.py

```diff
@@ -6,12 +8,16 @@
 from django.shortcuts import render
 from django.template.loader import render_to_string
 from django.utils.html import escape
+from django.utils.decorators import method_decorator
 from django.views.decorators.csrf import csrf_protect
-from django.views.decorators.http import require_POST
+from django.views.generic.edit import FormView
+from django.shortcuts import render, resolve_url
+
+from ..compat import url_has_allowed_host_and_scheme
```

Comment:
everywhere else we use absolute imports, I think we should use an absolute import here too.